src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -78,7 +77,8 @@ const DownloadReportPage = ({ runningUnitTest }) => {
   const fullState = useSelector(state => state);
 
   // Extract user info
-  const name = formatName(userProfile.userFullName);
+  const firstName = userProfile.userFullName.first;
+  const lastName = userProfile.userFullName.last;
```

Comment:
The `formatName` function also handled middle name and suffix. I would check with UCD and see if we need to print those as well. Also, check with them to see if first name could every be missing. If so, we'll need to handle it.

If you need to make any of these modifications, I would create a local `formatNameFirstLast` function right here to handle all of that logic, then just pass a single `name` variable into the PDF creator.

Reply:
Done